community/cypher/compatibility-suite/src/test/resources/org/neo4j/cypher/bdd/SimpleMatch.feature

```diff
@@ -0,0 +1,8 @@
+Feature: The First
+
+  Scenario: Simple match
+    Given init: CREATE ({name: "apa"});
```

Comment:
I know we think `apa` is a great name, but if this should be an official spec we should probably think of something better…